CHANGELOG.md

```diff
@@ -8,7 +8,8 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
-* None.  
+* Add method for formatting licenses for acknowledgements generation.    
```

Comment:
sorry, one more thing, this needs to have 2 empty spaces at the end for proper markdown formatting. I think also if possible add a link to this pull request as the other entries do. Thanks!

Reply:
Should be good now